examples/experimental/trial/unets_tf_keras/README.md

```diff
@@ -0,0 +1,19 @@
+*This folder contains the required files and the example code to use Tensorflow's Image Segmentation via UNet tutorial with Determined.*
+## The file version can be found on [Tensorflow Image Segmentation with UNet](https://www.tensorflow.org/tutorials/images/segmentation)
+
+For this implementation, we removed the system configuration since Determined will be managing this. The core functionality of the original script remains unchanged.
+
+### Folders and Files
+* **model_def.py**: Contains the core code for the model. This includes building and compiling the model.  
+* **const.yaml**: Contains the configuration for the experiment. This is also where you can set the flags used in the original script.  
+* **distributed.yaml**: Contains the configuration for a distributed experiment. This is also where you can set the flags used in the original script.  
+
```

Comment:
We should also perhaps add startup-hook.sh to this list just for completeness. Something like:

**startup-hook.sh**: Contains extra dependencies that Determined is required to install.